frontends/precinct-scanner/src/app_root.tsx

```diff
@@ -459,6 +458,10 @@ export function AppRoot({
           unconfigureMachine={() =>
             unconfigureServer({ ignoreBackupRequirement: true })
           }
+          showResetPollsToPausedButton
```

Comment:
why do we have this boolean if its just always true?

Reply:
It's not true on VxCentralScan, where I think this screen is also used.

Reply:
semi seperately should we also not show the button at all if the polls aren't in polls_closed_final?

Reply:
Right now it's just disabled if the polls aren't in `polls_closed_final`, I think that's fine for now. I actually originally had it such that it was hidden if polls were not in `polls_closed_final` but React / the testing didn't like how it was being removed the the page (trying to close the modal after the higher level state had already removed it from the page).